config/vite.json

```diff
@@ -0,0 +1,18 @@
+{
+  "all": {
+    "sourceCodeDir": "app/frontend",
+    "entrypointsDir": "entrypoints",
+    "watchAdditionalPaths": [],
+    "additionalEntrypoints": ["node_modules/govuk-frontend/govuk/assets/**/*"],
```

Comment:
Should the components css/js be included here?

Reply:
Not as entrypoints - they're already included as long as they're being imported in the respective css/js entrypoints.